tests/unit/fs/test_dvc.py

```diff
@@ -10,6 +10,20 @@
 from dvc.utils.fs import remove
 
 
+@pytest.mark.parametrize(
+    "path, key",
+    [
+        ("", ("repo",)),
+        (".", ("repo",)),
+        ("foo", ("repo", "foo")),
+        (os.path.join("dir", "foo"), ("repo", "dir", "foo")),
+    ],
+)
+def test_get_key(tmp_dir, dvc, path, key):
+    fs = DvcFileSystem(repo=dvc)
+    assert fs._get_key(path) == key
```

Comment:
As discussed before, a bit of an overkill to use real dvc in unit tests, but still kept using them in this one for simplicity. #7408 is a start to moving to fsspec, where we will scrap most of these tests for something better.